app/policies/podcast_import_policy.rb

```diff
@@ -1,4 +1,12 @@
 class PodcastImportPolicy < PodcastPolicy
+  def index?
+    PodcastPolicy.new(token, resource.podcast).new?
```

Comment:
`PodcastPolicy` doesn't have an `index?` method, but i figured `new?` was the scope that i wanted here.